<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="AU20R6LX1" name="com.ekaterina.HomePageTest" tests="4" failures="1" timestamp="10 Oct 2018 02:49:18 GMT" time="16.051" errors="0">
  <testcase name="navigToBankRate" time="0.000" classname="com.ekaterina.HomePageTest">
    <skipped/>
  </testcase> <!-- navigToBankRate -->
  <testcase name="openBrowser" time="16.051" classname="com.ekaterina.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [Free Credit Score &amp; Reporting + Personalized Offers | Bankrate] but found [Mortgage Calculator]">
      <![CDATA[java.lang.AssertionError: expected [Free Credit Score & Reporting + Personalized Offers | Bankrate] but found [Mortgage Calculator]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.ekaterina.BankRateTest.navigToBankRate(BankRateTest.java:39)
at com.ekaterina.HomePageTest.openBrowser(HomePageTest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- openBrowser -->
  <testcase name="navigatetocalc" time="0.000" classname="com.ekaterina.HomePageTest">
    <skipped/>
  </testcase> <!-- navigatetocalc -->
  <testcase name="enterLogIn" time="0.000" classname="com.ekaterina.HomePageTest">
    <skipped/>
  </testcase> <!-- enterLogIn -->
</testsuite> <!-- com.ekaterina.HomePageTest -->
